pkg/processor/build/builder.go

```diff
@@ -1893,3 +1896,19 @@ func (b *Builder) resolveNodeSelector(ctx context.Context) (map[string]string, e
 	return builderNodeSelector, nil
 
 }
+
+// resolveResources creates new limits containing the GPU-related limits from the function's limits, mapping each to zero.
+//
+//	By setting these limits to zero, the pods receive the necessary tolerations from the cloud provider for scheduling,
+//	without actually consuming GPU resources.
+func (b *Builder) resolveResources() v1.ResourceRequirements {
+	resources := v1.ResourceRequirements{
+		Limits: v1.ResourceList{},
+	}
+	for name := range b.options.FunctionConfig.Spec.Resources.Limits {
+		if strings.Contains(name.String(), "/gpu") {
```

Comment:
I see there's also a `-gpu` resource we look for.
It is probably quite old (alpha kubernetes) but better to make sure what are the supported gpu limit keys.